apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -77,6 +78,7 @@ public static Builder builder() {
   private final CacheControl defaultCacheControl;
   private final CacheHeaders defaultCacheHeaders;
   private final ApolloLogger logger;
+  private final ApolloCallTracker callTracker = new ApolloCallTracker();
```

Comment:
I am wondering how we are going to lazily create it. If we had a builder method to supply the `IdleCallback` then it would have been possible, but I have already expressed my concern over why we shouldn't take that route. 

Or maybe if we are thinking of creating the ApolloTracker when someone sets the idleCallback, then I am afraid that wouldn't give us an exact count of running calls e.g. if an async call is made and then an idleCallback is set, then the apolloTracker would give us 0 running calls even though we have a running call. 

Do you have anything else in your mind?